pkg/actuators/awsplacementgroup/controller_suite_test.go

```diff
@@ -1,61 +0,0 @@
-package awsplacementgroup
```

Comment:
Does this need to be deleted?

Reply:
Now that we reintroduced the types, we can bring this back.